src/components/Chart/index.tsx

```diff
@@ -0,0 +1,67 @@
+'use client'
+
+import React, { useState, useEffect } from 'react'
+import { Bar, Line } from 'react-chartjs-2'
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  BarElement,
+} from 'chart.js'
+
+ChartJS.register(CategoryScale, LinearScale, BarElement)
+
+const options = {
+  responsive: true,
+  plugins: {
+    legend: {
+      display: false,
+    },
+  },
+  scales: {
+    x: {
+      grid: {
+        display: false, // Removes the background grid lines for X-axis
+      },
+    },
+    y: {
+      display: false, // Removes the numbers for the y-axis
+    },
+  },
+}
+
+interface ChartDataInterface {
+
+}
```

Comment:
this is not used

Reply:
It's prepped bc idk how my data is going to look just yet. I haven't passed it to the chart component. I just prepped it.